test/source/mock/lib/api.ts

```diff
@@ -144,19 +146,24 @@ export class Api<REQ, RES> {
     if (!req.url) {
       throw new Error('no url');
     }
-    if (this.handlers[req.url]) {
+    const attesterHandler = getMockAttesterEndpoints(this.attesterConfig);
+    const newHandlers: Handlers<REQ, RES> = {
+      ...(attesterHandler as Handlers<REQ, RES>),
+      ...this.handlers,
+    };
```

Comment:
Can we move `getMockAttesterEndpoints` to [all-apis-mock.ts](https://github.com/FlowCrypt/flowcrypt-browser/blob/master/test/source/mock/all-apis-mock.ts#L29) where all mock endpoints are listed?

Reply:
IMO, we can't as we have to use live `Api.attesterConfig` variable in `getMockAttesterEndpoints`

Reply:
Yeah, you're right 👍
But let's rename `newHandlers` to `allHandlers`, I think it'll be more appropriate name